### PR TITLE
Add kyverno sample policy

### PR DESCRIPTION
**Pull request by jianh619**

### CODE REVIEW

File: community/CM-Configuration-Management/policy-kyverno-sample.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-kyverno-sample
+  annotations:
+    policy.open-cluster-management.io/standards: NIST-CSF
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-kyverno-sample.yaml]: @jianh619 can you update the standard/catogery/control to NIST 800-53 see https://github.com/open-cluster-management/policy-collection/blob/master/community/CM-Configuration-Management/policy-gatekeeper-sample.yaml#L6-L8

File: community/CM-Configuration-Management/policy-kyverno-sample.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-kyverno
+        spec:
+          remediationAction: inform
+          severity: low
+          namespaceSelector:
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-kyverno-sample.yaml]: namespaceSelector can be omitted as ClusterPolicy is cluster level resource

**Comment by ycao56** [comment]
Thanks @jianh619 ! This is really great. Just some minor comments. @dockerymick can you review the text?

File: community/README.md [hunk 3]
```diff
@@ -84,6 +84,7 @@ Policy  | Description | Prerequisites
...
 [Integrity Shield Events](./CM-Configuration-Management/policy-integrity-shield-events.yaml)| Use the Integrity Shield Events policy to show a status, which represents whether Integrity Shield has denied some requests in a cluster or not. | See the [Integrity Shield documentation.](https://github.com/open-cluster-management/integrity-shield/blob/master/docs/ACM/README_SHOW_INTEGRITY_STATUS_ON_ACM_UI.md)
+[Kyverno sample policy](./CM-Configuration-Management/policy-kyverno-sample.yaml) | Use the Kyverno sample policy to view an example of how kyverno policy can be applied to a managed cluster , this policy will be evaluate by kyverno controller on managed cluster .| See the [Kyverno documentation](https://kyverno.io/docs/writing-policies) for how to write policy. **Note**: Kyverno must be installed on managed cluster to use Kyverno policy.
```
Comment (dockerymick) [inline community/README.md]: I suggest the following update:

Use the Kyverno sample policy to view an example of how a kyverno policy can be applied to a managed cluster. This policy is evaluated by the kyverno controller on a managed cluster.



Kyverno must be installed on a managed cluster to use Kyverno policy.

**Review by dockerymick — COMMENTED** [review]
Thanks for adding the description, I made a comment. Otherwise this looks good

File: community/README.md (hunk 3 above)
Comment (ycao56) [reply community/README.md]: I suggest also add a link to https://kyverno.io/docs/installation/#install-kyverno-using-yamls for installation instructions

**Comment by jianh619** [comment]
@ycao56 @dockerymick Thanks for your time , just back from holiday , I modified the code following your suggestions and  rebased it to resolve conflict , that's why you saw more comments in this PR .

**Review by dockerymick — APPROVED** [review]
Made a small update, the PR looks good!

**Comment by ycao56** [comment]
@jianh619 your branch seems out of date. There are a lot of file changes unrelated to your pr. Can you squash the commits and rebase?

**Comment by jianh619** [comment]
@ycao56  modified as request , could you help to review please , thanks

**Review by ycao56 — APPROVED** [review]

**Pull request merged**